Cargo.toml

```diff
@@ -10,15 +10,19 @@ license = "MIT"
 readme = "README.md"
 
 [dependencies]
-embedded-hal = { version = "0.2.3", features = ["unproven"] }
-either = { version = "1.5.3", default-features = false }
+embedded-hal = { version = "0.2.4", features = ["unproven"] }
+either = { version = "1.6.1", default-features = false }
 
 [dev-dependencies]
-version-sync = "0.8"
-cortex-m = "0.6.1"
-cortex-m-rt = "0.6.10"
+version-sync = "0.9.1"
+cortex-m = "0.6.3"
```

Comment:
Shall we just go with `0.6` for both? I'm not sure why I specified patch version numbers oiriginally, it's pretty unnecessary. Same with the `embedded-hal` and `either` versions if you wouldn't mind

Reply:
Agree. I will push this with the other changes into this branch.